eng/ci/templates/official/jobs/build-extension-artifacts.yml

```diff
@@ -1,10 +1,13 @@
 parameters:
+  - name: JobName
+    type: string
+    default: BuildArtifacts
   - name: ExtensionProjectName
     type: string
 
 jobs:
 
-- job: BuildArtifacts
+- job: ${{ parameters.JobName }}
```

Comment:
You could also calculate the job name from `ExtensionProjectName`

```suggestion
- job: Build_${{ parameters.ExtensionProjectName }}
```

Reply:
I don't think `.` are supported in job names?

> name of the job, A-Z, a-z, 0-9, and underscore

Reply:
Taking a different approach, this wont be required